src/ucp/wireup/select.c

```diff
@@ -345,9 +346,10 @@ ucp_wireup_select_transport(const ucp_wireup_select_params_t *select_params,
      * best one has the highest score (from the dedicated score_func) and
      * has a reachable tl on the remote peer */
     ucs_for_each_bit(rsc_index, tl_bitmap) {
-        resource   = &context->tl_rscs[rsc_index].tl_rsc;
-        iface_attr = ucp_worker_iface_get_attr(worker, rsc_index);
-        md_attr    = &context->tl_mds[context->tl_rscs[rsc_index].md_index].attr;
+        resource            = &context->tl_rscs[rsc_index].tl_rsc;
+        iface_attr          = ucp_worker_iface_get_attr(worker, rsc_index);
```

Comment:
do we still use iface_attr? why need both iface_attr and adjusted_iface_attr?

Reply:
Not sure if iface_attr was used elsewhere but wanted to leave it unaltered in case the original values were important.